terraform-plans/templates/github/charm_release.yaml.tftpl

```diff
@@ -19,25 +19,25 @@ jobs:
   release:
     needs: check
     runs-on: $${{ matrix.runs-on }}
-    strategy:
-      fail-fast: false
-      matrix:
-        runs-on: ${runs_on}
     steps:
       - name: Checkout
         uses: actions/checkout@v4
         with:
           submodules: true
 
-      - name: Initialize lxd  # This should dropped once it's implemented on charming-actions itself. https://github.com/canonical/charming-actions/issues/140
-        # revision is latest main at time of writing; using because it contains a fix to https://github.com/canonical/setup-lxd/issues/19
-        uses: canonical/setup-lxd@2aa6f7caf7d1484298a64192f7f63a6684e648a4
+      - name: Download the tested charms
+        uses: actions/download-artifact@v4
+        with:
+          pattern: built_charms_*
+          merge-multiple: true
+
+      - name: List the names of the tested charms
+        run: |
+          echo "CHARM_NAMES=$(ls *.charm | paste -sd ,)" | tee -a "$GITHUB_ENV"
 
       - name: Pack and upload to charmhub
-        uses: canonical/charming-actions/upload-charm@2.6.2
+        uses: canonical/charming-actions/upload-charm@2.7.0
         with:
-          charmcraft-channel: "${charmcraft_channel}"
           credentials: "$${{ secrets.CHARMHUB_TOKEN }}"
           github-token: "$${{ secrets.GITHUB_TOKEN }}"
-          # Ensure the charm is built in an isolated environment and on the correct base in an lxd container.
-          destructive-mode: false
```

Comment:
It's not available in the new build procedure anymore. Why don't we need it anymore?

Reply:
because we don't build anymore

Reply:
We have `build` process in  [charm_check.yaml.tftpl](https://github.com/canonical/solutions-engineering-automation/pull/161/files#diff-97ae0cd1ad9d06cfe014821942219ccf29e0797bdc5c3c3cd3067cfa6372499d). Should we disable/enable destructive-mode ?

Reply:
`charmcraft pack` by default runs in **non** destructive mode

Reply:
Thanks! I find the document in 
https://github.com/canonical/craft-actions?tab=readme-ov-file#charmcraftpack